themes/default/content/signup/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Get started with Pulumi
+meta_desc: Pulumi is an infrastructure as code tool that enables engineers to write infrastructure code in a language they already know and love. Get started for free.
+meta_image: 
```

Comment:
If we don't end getting a meta image before merging, I'd just remove this field so it uses the default image.

Reply:
do we just need the meta image to say "get started with pulumi"?

Reply:
We're going to go with the default image, I'll just delete the field